From 3f7b6a391127b7a0eb4dc566e4115373a66bbd6c Mon Sep 17 00:00:00 2001
From: Konstantin Ryabitsev <konstantin@linuxfoundation.org>
Date: Fri, 2 Jan 2026 15:25:57 -0500
Subject: Fix typo in classifier

This is only relevant to PyPI, so this can live post-tag.

Signed-off-by: Konstantin Ryabitsev <konstantin@linuxfoundation.org>
---
 pyproject.toml | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/pyproject.toml b/pyproject.toml
index 25592ac..a541b01 100644
--- a/pyproject.toml
+++ b/pyproject.toml
@@ -14,7 +14,7 @@ readme = "README.rst"
 requires-python = ">=3.8"
 keywords = ["git", "patches", "attestation", "email", "workflow"]
 classifiers = [
-    "Intended Auidience :: Developers",
+    "Intended Audience :: Developers",
     "Topic :: Software Development :: Libraries :: Python Modules",
     "Topic :: Communications :: Email",
     "Topic :: Software Development",
-- 
cgit 1.2.3-korg

